### DISCUSSION TITLE
Welcome to disctopia-c2 Discussions!

### DISCUSSION THREAD

**Discussion by 3ct0s** [3ct0s/disctopia-c2#113] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hanar1988** [comment]
i have an issue in wine on raspberry-pi 
 wine: failed to open "/root/.wine/drive_c/users/root/AppData/Local/Programs/Python/Python38-32/Scripts/pyinstaller.exe": c0000135

**Reply by tapha7** [reply]
same issue on kali linux run from wsl
